src/controllers/artists.js

```diff
@@ -0,0 +1,55 @@
+const Artist = require('../models/artist');
+const { find } = require('mongoose');
+
+exports.create = (req, res) => {
+  const artist = new Artist({
+    name: req.body.name,
+    genre: req.body.genre,
+  });
+
+  artist.save().then(() => {
+    res.status(201).json(artist);
+  });
+};
+
+exports.list = (req, res) => {
+  Artist.find({}, (err, artists) => {
+    res.status(200).send(artists);
+  });
+};
+
+exports.find = (req, res) => {
+  Artist.findById(req.params.artistId, (err, Aid) => {
```

Comment:
`Aid` is a bit obscure, it's also not a class so shouldn't be capitalised. I would probably just stick with `artist` here.